var/lib/istio/config/mesh

```diff
@@ -0,0 +1,110 @@
+# Default config. Will be included in the image/deb.
```

Comment:
can we make this `mesh.yaml`?

Reply:
Current installer creates a file called 'mesh' - I don't know yet if it'll help for the migration.

Easy to mount it under /var/lib/istio insted of /etc - but the file name is not.

Reply:
I think if we implement a merge - we can rename. 
The order could be:
- read /var/lib/istio/cluster/mesh -> the 'installer' generated mesh
- read /var/lib/istio/config/mesh.yaml -> user overrides or local

But later.